packages/core/src/components/annotation/annotation.scss

```diff
@@ -4,11 +4,11 @@
 @use "~dso-toolkit/src/components/insert";
 @use "~dso-toolkit/src/components/delete";
 
-@use "~dso-toolkit/src/components/insert/insert" as *;
-@use "~dso-toolkit/src/components/delete/delete" as *;
+@use "~dso-toolkit/src/components/anchor/anchor";
 
 :host {
   display: block;
+  padding: 2px;
```

Comment:
Niet `units.$u1 * 0.25` 🤣

Reply:
Tsja, lastig he. Het ontwerp geeft expliciet 2px. Het acteert als border, dus ik zou ook iets als `$border-width * 2` kunnen doen.

Of een commentaartje erbij. Het 8px grid is wat dat betreft ook maar een richtlijn.

Reply:
We laten 'm zo.

Groet,
Eric & Thomas